hooks/hookexecution/executor_test.go

```diff
@@ -760,7 +760,7 @@ func TestExecuteProcessedAuctionStage(t *testing.T) {
 		{
 			description:            "Stage execution can be rejected - and later hooks rejected",
 			givenPlanBuilder:       TestRejectPlanBuilder{},
-			givenAccount:           nil,
+			givenAccount:           account,
```

Comment:
We need to pass account here, because we want to test that BidRequest will not change even if we restrict activities: `getModuleActivities("foo", false, false)` and payload for the hook will be modified.